style.css

```diff
@@ -104,6 +104,222 @@
   list-style: none;
 }
 
+/* work-section */
+
+.work-section {
+  width: 100%;
+  height: 2788px;
+  top: 718px;
+}
```

Comment:
- [ ] Please make sure to use `display: grid` for the main container of the projects as mentioned in the project requirements 📌 Following is a quote for a related requirement 👇

  > _Use CSS Grid in the main container that contains all of the cards_


- [ ] Please note that the `background-color` of the works container is `#F7F7F9`. Please fix this ⛏
- [ ] Also, please note that there is some space/margin between the `headline` section and `works` section. Kindly implement that as well. See the images below for more clarity 💡


Your design | Figma design
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235183054-f31511bf-f8a0-473d-99e8-1f2574372d23.png)| ![image](https://user-images.githubusercontent.com/106119105/235182942-c80458f2-314b-413a-a642-2359c39c68b0.png)


- [ ] I noticed that there is a lot of space between `works` section and `about` section. Please fix this according to the design guidelines 📂 _Hint: This is related to height_


Your design | Figma design
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235191618-da85c9a8-3568-4bbd-a699-0d7d3f60b621.png) | ![image](https://user-images.githubusercontent.com/106119105/235191545-55e247d4-bf0a-4266-a0f1-985ac32b457c.png)